test/e2e/lib/pages/signup/pick-a-plan-page.js

```diff
@@ -77,4 +84,29 @@ export default class PickAPlanPage extends AsyncBaseContainer {
 			await driverHelper.clickWhenClickable( this.driver, selector );
 		}
 	}
+
+	async scrollPlanInToView( level ) {
```

Comment:
We need to be aware of this - it may happen it will break at this point if something is changed in plans order.